src/Simulation/NativeSparseSimulator/test.ps1

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Write-Host "##[info]Test Native Sparse Simulator"
+Push-Location (Join-Path $PSScriptRoot "build")
+
+ctest -C "$Env:BUILD_CONFIGURATION" --verbose
+
+if ($LastExitCode -ne 0) {
+    Write-Host "##vso[task.logissue type=error;]Failed to test Native Sparse Simulator"
+    $script:all_ok = $False
```

Comment:
It looks like this variable hasn't been defined or used elsewhere in the script?

Reply:
Fixed.